### ISSUE TITLE
Some deprecation warnings not compatible with codecs

### ISSUE THREAD

**Issue by stambata** [softwaregroup-bg/ut-error#10]
E.g. codes which start capital letters and numbers are used as error types.
https://github.com/softwaregroup-bg/ut-codec-payshield/blob/master/errors/list.json

**Comment by stambata** [comment]
@kalinkrustev @zetxx

**Comment by zetxx** [comment]
yep, unfortunately we use such things in payshield, cause we have a list of errors by number (byte maybe: 00-FF)
